### ISSUE TITLE
Sounds still trigger on items hidden by Ground Items' "only show loot" toggle

### ISSUE THREAD

**Issue by XaraVFX** [ThePharros/ground-item-sounds#2]
When using the `only show loot` toggle in the ground items plugin to hide loot from others, their item drops will still trigger a sound effect regardless. It would be nice to have this prevented by default as I personally can't think of any use cases where the current behavior is preferred

**Comment by widget-** [comment]
Would subscribing to `onNpcLootReceived` instead of `onItemSpawned` fix this?

That seems to be what the GroundItems plugin is doing:

https://github.com/runelite/runelite/blob/f613939/runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java#L305-L310

**Comment by kulcris** [comment]
Seconded.

**Comment by RB490** [comment]
Yes! I love this plugin but the false positives caused by this are really bothersome

**Comment by ThePharros** [comment]
Sorry i've been MIA with RL dev stuff. I'll look into this this week.

**Comment by ThePharros** [comment]
This issue should now be fixed once RL approves the update.

**Comment by RB490** [comment]
> This issue should now be fixed once RL approves the update.

Appreciated!
